ref/network-ports.md

```diff
@@ -0,0 +1,35 @@
+
+<a name="anbox-cloud"></a>
+## Anbox Cloud
+
+| Service               | Port(s)     | Protocol | Exposed externally | Required                     | Description               |
+|-----------------------|-------------|----------|--------------------|------------------------------|---------------------------|
+| AMS                   | 8444        | TCP      | no                 | yes                          | HTTPS API                 |
+| AMS                   | 20002       | TCP      | no                 | no                           | HTTPS Prometheus endpoint |
+| AMS node controller   | 10000-11000 | TCP      | yes                | no                           | Container service ports   |
+| Anbox Cloud Dashboard | 443         | TCP      | no                 | no                           | HTTPS website             |
+| Anbox Stream Agent    | 443         | TCP      | no                 | yes                          | HTTPS API                 |
+| Anbox Stream Gateway  | 4000        | TCP      | no                 | yes                          | HTTPS API                 |
+| Anbox Stream Gateway  | 7000        | TCP      | no                 | yes                          | Dqlite HA and API         |
+| Coturn                | 5349        | TCP      | yes                | yes                          | STUN/TURN                 |
+| Coturn                | 5349        | UDP      | yes                | yes                          | STUN/TURN                 |
+| Coturn                | 50000-51000 | UDP      | yes                | no (unless TURN is required) | TURN relay ports          |
+| etcd                  | 2379        | TCP      | no                 | yes                          | gRPC API                  |
+| LXD                   | 8443        | TCP      | no                 | yes                          | HTTPS API                 |
+| NATS                  | 4222        | TCP      | no                 | yes                          | API                       |
+
+
+<a name="appliance"></a>
+## Anbox Cloud Appliance
+
+| Service               | Port(s)     | Protocol | Required | Description                           |
+|-----------------------|-------------|----------|----------|---------------------------------------|
+| AMS node controller   | 10000-11000 | UDP      | no       | Container service ports               |
+| AMS node controller   | 10000-11000 | TCP      | no       | Container service ports               |
+| Anbox Cloud Dashboard | 80          | TCP      | yes      | HTTP (redirects to HTTPS on port 443) |
+| Anbox Cloud Dashboard | 443         | TCP      | yes      | HTTPS website                         |
+| Coturn                | 5349        | UDP      | yes      | STUN/TURN                             |
+| Coturn                | 60000-60100 | UDP      | yes      | TURN relay ports                      |
```

Comment:
These ports are different than the ones for the full deployment - correct?

Reply:
Yes, we expose less ports than with the full deployment due to limitations in NFT rules LXD generates (1000 ports are too much for it)